src/test/integration/portfolio-item-integration.test.js

```diff
@@ -356,7 +356,7 @@ describe('Integration tests for portfolio items', () => {
       .onGet(
         `${CATALOG_API_BASE}/portfolio_items/source-offering-1/next_name?destination_portfolio_id=123`
       )
-      .replyOnce(200, { next_name: `Copy of ${addedPortfolioItem.name}` });
+      .reply(200, { next_name: `Copy of ${addedPortfolioItem.name}` });
```

Comment:
Is `mock.reset();` called after this? I can imagine that changing from `once` could affect other test files.

Reply:
It won't since this endpoint must always return the same value. Now its called multiple times since it's handled by the "spy" component.